tests/D2L.CodeStyle.Analyzers.Test/UnsafeStatics/UnsafeStaticsAnalyzerTests.cs

```diff
@@ -687,12 +765,16 @@ class Safe : IUnsafe {
 				public static readonly Safe Instance { get; } = new Safe();
 			}
 
-			private readonly static IUnsafe foo = Safe.Instance;
+			private readonly static IUnsafe foo = Safe.Instance; // bad, Safe is not sealed

```

Comment:
note: `foo` is actually safe here, _because_ `Safe.Instance` is safe. But we don't currently expose other members' inspections (maybe we could via Roslyn annotations), soooo we don't allow it.

If you're thinking that adding this feature would make some types of errors go away (`ConstructorBinder<T>.AllFields`), unfortunately it won't, because we need source code for that level of analysis, which we don't have because of non-project references.